x-pack/tasks/build.js

```diff
@@ -56,5 +56,21 @@ export default (gulp, { buildTarget }) => {
         directory: buildRoot
       })
     );
+
+    // removes Kibana packages to allow for installation of dependencies
+    // which are not included in the Yarn workspace
+    const kibanaRoot = resolve(__dirname, '../..');
+    const projectPaths = getProjectPaths(kibanaRoot);
+    const projects = await getProjects(kibanaRoot, projectPaths);
+    const pkg = JSON.parse(readFileSync(resolve(buildRoot, 'package.json')));
+
+    pkg.dependencies = Object.keys(pkg.dependencies)
+      .filter((d) => !projects.has(d))
```

Comment:
I am thinking we could additionally remove any dependencies with the same version qualifier as the dependency in Kibana.

Reply:
can you add any context on what's happening?  wouldn't yarn workspace dedupe these to the top level automatically?

Reply:
oh, this is outside the scope of kbn-pm et all as a side effect